### PR TITLE
Hooli-style Popup

### PR DESCRIPTION
**Pull request by natalia-artsiukh**
### Hooli-style Popup

[Demo](https://natalia-artsiukh.github.io/kottans_homework1/) |
[Code base](https://github.com/natalia-artsiukh/kottans_homework1.git)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by kasionio** [comment]
Great job! Nice formatting, which is pretty easy to read, that's cool. The menu is also looking good 

Seems there is still some room for UI and accessibility improvements.
1. I can't reach the popup menu using just a keyboard. 
2. Hover effect "cursor: pointer" should appear only on clickable elements(buttons, links, etc.), not on the whole container.
3. "More" button should take 100% width of the popup container by design, seems more containers will be needed in code for this :)
4. Please take a look at the screenshot. Seems that the container needs to be better centered and also should have just one padding, the second looks redundant.
5. To prevent jumping on elements if you want to add some padding on hover can be done like so:
`
.element {
padding: 1px solid transparent;
}
.element: hover {
padding: 1px solid SOME_COLOR;
} 
`

image

Don't hesitate to ask if something is not clear ;)

**Comment by natalia-artsiukh** [comment]
I made the necessary changes to my files. please check)
I couldn't make a re-request review because I don't have reviewers

File: submissions/natalia-artsiukh/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                        </a>
+                        <a href="#" class="popup-item">
+                            <img src="./img/photos.png" alt="photos" class="popup-img">
+                            <p class="popup-text">Photo</p>
+                        </a>
+                        <a href="#" class="popup-item">
+                            <img src="./img/apps.png" alt="apps" class="popup-img">
+                            <p class="popup-text">Apps</p>
+                        </a>
+                        <a href="#" class="popup-item">
```
Comment (A-Ostrovnyy) [inline submissions/natalia-artsiukh/html-css-popup/index.html:66]: Fix focus outline
image

File: submissions/natalia-artsiukh/html-css-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Popup</title>
+</head>
+<body>
+    <header class="header">
+        <nav class="header_nav">
+            <div class="nav-position">
```
Comment (A-Ostrovnyy) [inline submissions/natalia-artsiukh/html-css-popup/index.html:12]: For menu items especially if you have nested navigation use an unordered list.

File: submissions/natalia-artsiukh/html-css-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+                <a href="#" class="link">Images</a>
+                <input type="checkbox" class="check-input" id="check">
+                <label for="check" class="label-position">
+                    <span class="box-popup"></span>
+                </label>
+                <div class="popup">
+                    <div class="popup-angle"></div>
+                    <div class="popup-content">
+                        <a href="#" class="popup-item">
+                            <img src="./img/calendar.png" alt="calendar" class="popup-img">
```
Comment (A-Ostrovnyy) [inline submissions/natalia-artsiukh/html-css-popup/index.html:23]: Read [this article](https://htmlacademy.ru/blog/articles/alt-text) about `alt` attribute. Sorry for rusisan but it's a good article.

File: submissions/natalia-artsiukh/html-css-popup/style.css [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+}
+.profile-link {
+    overflow: hidden;
+}
+.profile-img {
+    display: block;
+    background-color: rgba(255, 235, 205, 0.614);
+    width: 100%;
+}
+@media (width: 1920px) {
```
Comment (A-Ostrovnyy) [inline submissions/natalia-artsiukh/html-css-popup/style.css:164]: Don't forget about mobile view 
image

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
See my comments below.

File: submissions/natalia-artsiukh/html-css-popup/style.css [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+.popup-items {
+    padding: .5em 1.5em 0;
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, 2fr);
+    justify-items: center;
+    align-items: center;
+    gap: .5em;
+    border-top: 1px solid rgba(29, 39, 41, 0.658);
+    display: none;
```
Comment (kasionio) [inline submissions/natalia-artsiukh/html-css-popup/style.css:148]: two `display` is too much for one class :)

File: submissions/natalia-artsiukh/html-css-popup/style.css [hunk 6]
```diff
@@ -0,0 +1,149 @@
+body {
+    margin: 0;
+    padding: 0;
+    background-image: url(./img/ocean.jpg);
+    background-repeat: no-repeat;
+    box-sizing: border-box;
+}
+.header {
+    background-color: rgba(110, 120, 116, 0.598);
```
Comment (kasionio) [inline submissions/natalia-artsiukh/html-css-popup/style.css:8]: Why do you need so much accuracy on alpha channel value?

File: submissions/natalia-artsiukh/html-css-popup/style.css [hunk 7]
```diff
@@ -0,0 +1,149 @@
+body {
+    margin: 0;
+    padding: 0;
+    background-image: url(./img/ocean.jpg);
```
Comment (kasionio) [inline submissions/natalia-artsiukh/html-css-popup/style.css:4]: you can use `background` shorthand here

File: submissions/natalia-artsiukh/html-css-popup/style.css [hunk 8]
```diff
@@ -0,0 +1,149 @@
...
+.nav-position {
+    display: flex;
+    justify-content: space-evenly;
+    width: 30%;
+    align-items: center;
+    line-height: 3em;
+}
+.link {
+    text-decoration: none;
+    color: blanchedalmond;
```
Comment (kasionio) [inline submissions/natalia-artsiukh/html-css-popup/style.css:23]: Better to use HEX or RGB model

File: submissions/natalia-artsiukh/html-css-popup/style.css [hunk 9]
```diff
@@ -0,0 +1,149 @@
...
+    font-size: 18px;
```
Comment (kasionio) [inline submissions/natalia-artsiukh/html-css-popup/style.css:24]: To use a responsive design, even when a page will be scaled you can use `em` or `rem`

**Review by kasionio — CHANGES_REQUESTED** [review]
Great job! Last steps left

**Review by A-Ostrovnyy — APPROVED** [review]
Looks good to me.
@kasionio what's your input about it?

**Comment by kasionio** [comment]
@A-Ostrovnyy, agree. @natalia-artsiukh, congrats 🎉

**Review by kasionio — APPROVED** [review]

**Pull request merged**